Automattic/wp-calypso
# Split files with multiple @parallel tests

#### Background

Magellan will find all `describe` blocks tagged with `@parallel` and send each one to a different Mocha process to be run in parallel.

Parallelization in Jest is a bit different: it will send each _file_ to a separate process.

#### Changes proposed in this Pull Request

* To ease the migration Mocha->Jest, this PR splits files with mutiple `@parallel` blocks so Mocha behaviour matches Jest.

#### Testing instructions

* Verify Calypso e2e tests are passing
* Verify the number of tests is the same as `trunk`.
